### PR TITLE
history 5.2.7

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

Add 5.2.7 entry

### CODE REVIEW

File: history.txt [hunk 1]
```diff
@@ -1,8 +1,14 @@
 OMERO version history
 =====================
 
+5.2.6 (December 2016)
+---------------------
+
+This is a release aimed at system administrators or developers willing to
+build OMERO with Ice 3.6.3
```
Comment (mtbc) [inline history.txt]: end with period

File: history.txt [hunk 2]
```diff
@@ -1,8 +1,14 @@
 OMERO version history
 =====================
 
+5.2.6 (December 2016)
+---------------------
+
+This is a release aimed at system administrators or developers willing to
```
Comment (hflynn) [inline history.txt]: 'wanting' instead of 'willing'? Or are we recommending people use it?

Comment (jburel) [reply history.txt]: 3.6 is not the recommended version on 5.2.x
Will replace by "wanting"

Comment (hflynn) [reply history.txt]: 👍

**Comment by hflynn** [comment]
Perhaps we should clarify that no other fixes are included and specifically that BF support remains at version 5.1.10(?) since various people have been asking for us to support BF 5.2

**Comment by jburel** [comment]
@hflynn: that's certainly something we need to mention.
Thanks for the reminder

**Comment by jburel** [comment]
The B-F will need to be bumped to 5.1.10 it is currently at 5.1.9
I will add a note about B-F version in history

**Comment by hflynn** [comment]
👍

**Comment by jburel** [comment]
see https://github.com/openmicroscopy/openmicroscopy/pull/4978

**Comment by hflynn** [comment]
If I was being really nitpicky, I'd say it would be better to have `5.1.10` as the link text for the hyperlink to the BF version history and that we don't usually write OME Model with a hyphen but neither of these are blockers to getting this and the accompanying docs PR merged as staging looks fine as is.

**Comment by jburel** [comment]
done

**Comment by hflynn** [comment]
Good to merge

**Pull request merged**